utils.py

```diff
@@ -1081,8 +1133,8 @@ def self_update():
     if now >= check_again:
         logging.debug("Running self-update.")
 
+        set_logoslinuxinstaller_latest_release_config()
         set_recommended_appimage_config()
```

Comment:
In testing with `-uD` I see where this function `check_for_updates` runs, but I was expecting that sometime afterwards it would call `utils.update_to_latest_lli_release` to actually perform the update. I'm not seeing this happen. Am I making a bad assumption or missing something?

Reply:
I see what's going on. My test is this: Logos not installed, run `./LogosLinuxInstaller.py -auD`. This *should* run the `update_self` action, except there's a mismatch between the function name to be run and the name listed in the `install_not_required` list. I'll add a more detailed comment in that part of the code.